src/components/ButtonWithDropdownMenu/index.tsx

```diff
@@ -73,6 +73,9 @@ function ButtonWithDropdownMenu<IValueType>({
                         success={success}
                         pressOnEnter={pressOnEnter}
                         ref={(ref) => {
+                            if (isSplitButton) {
+                                return;
+                            }
```

Comment:
Why is this needed?

Reply:
Okay can we rename `caretButton` to `dropdownAnchor` since it's not always assigned to the caret?